api/app/src/Controller/UserController.php

```diff
@@ -651,4 +651,25 @@ public function getPrimaryEmail(int $deputyUid): string
 
         return $userEmail;
     }
+
+    /**
+     * Endpoint for getting the primary user account associated with a deputy uid.
+     *
```

Comment:
As this is an admin action do we not want to limit this endpoint using a `@security` tag?

Reply:
So on the API side we don't tend to add the `@security` tag. The reason for this is that the requests made to these endpoints on the API side could be coming from a deputy request or an admin request (the API endpoints don't care).
However, they do have the some level of security checks and that's to do with the key that is passed in with the API request. It does this to make sure that the request is legitimate and has come from the frontend, and not from anywhere else.

If you look at `client/app/src/Controller/Admin/Client/ClientController.php` you'll notice that there is a `@security` tag on that `detailsAction` endpoint.